xml/schema/common.xml

```diff
@@ -99,7 +99,7 @@
 		</field>
 		<field name="installed_locales" type="C2" size="1024">
 			<NOTNULL/>
-			<DEFAULT VALUE="en_US"/>
```

Comment:
This file shouldn't be changed; it's historical, and will be used for pre-3.3 upgrades.